docs-next/pages/apis/config.mdx

```diff
@@ -160,7 +160,7 @@ export default config({
 import type { ServerConfig } from '@keystone-next/types';
 ```
 
-The **run** commands from the Keystone [command line](../guides/cli) will start an Express web-server for you.
+The `dev` and `start` commands from the Keystone [command line](../guides/cli) will start an Express web-server for you.
```

Comment:
We no longer frame the CLI in groups of commands, so **run** no longer means anything.